lib/presentation/pages/tip_resources_page.dart

```diff
@@ -0,0 +1,69 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_inappwebview/flutter_inappwebview.dart';
+import 'package:infinite_horizons/presentation/atoms/atoms.dart';
+import 'package:infinite_horizons/presentation/atoms/separator_atom.dart';
+import 'package:infinite_horizons/presentation/molecules/molecules.dart';
+
+class TipResourcePage extends StatelessWidget {
+  const TipResourcePage({
+    required this.url,
+    super.key,
+  });
+  final String url;
+
+  @override
+  Widget build(BuildContext context) {
+    if (url.endsWith(".pdf")) {
+      return Scaffold(
+        body: Column(
+          children: [
+            TopBarMolecule(
+              topBarType: TopBarType.back,
+              title: "Resource",
+              onTap: () => Navigator.pop(context),
+            ),
+            const SeparatorAtom(variant: SeparatorVariant.closeWidgets),
+            Expanded(
+              child: PdfViewerMolecule(
+                url: url,
+              ),
+            ),
+          ],
+        ),
+      );
+    } else if (url.contains("youtube") || url.contains("youtu.be")) {
+      return Stack(
+        children: [
+          GestureDetector(
+            onTap: () => Navigator.pop(context),
+            child: Container(
+              height: double.infinity,
+              color: Colors.transparent.withOpacity(0.5),
+            ),
+          ),
+          YoutubePlayerMolecule(
+            url: url,
+            hideThumbnail: true,
+          ),
+        ],
+      );
+    }
```

Comment:
I would like to see how this section is displayed out, can you share a screenshot
Maybe we want to add a Scaffold( with TopBarMolecule like we do for all pages to keep it consistent.

Reply:
![image](https://github.com/guyluz11/infinite_horizons/assets/114633095/f71ed362-1fcc-4f65-bc1f-13d34d020103)
Here is a sample image of how it looks, I've tried to replicate a simple popup

Reply:
Hmm it doesn't look good as a popup and the behavior should look consistent to opening pdf's.
So if pdf opens in a new page the video should do the same.
Let's move it to the video page.

Also is there a working full-screen button on the video player?, I think the player supports it by default and we just need to allow it.
It is a nice have, not mandatory.

Reply:
I have made changes to use the same page for YoutubePlayerMolecule but it's fullscreen doesn't interact so well with the TopBarMolecule 
![image](https://github.com/guyluz11/infinite_horizons/assets/114633095/5d2d1936-74e9-405a-9544-0a28f9aaccd4)

Reply:
So disable full screen for now

Reply:
You can't detect when the video got switched???, if it is really the case then this is a bad package and we should switch to the second one.

I have used some YouTube players before on Flutter and you definitely can listen to this in some of the players.

If you want me to take a look at it let me know.

Reply:
I will look into a different package